aws/handlers/users_api_req_handler.go

```diff
@@ -0,0 +1,31 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/sal-org/sal-backend/aws/database"
+	"github.com/sal-org/sal-backend/aws/repositories"
+	"github.com/sal-org/sal-backend/aws/utils"
+	"github.com/sal-org/sal-backend/models"
+)
+
+// HandlePatientsRequest will handle all users gate api requests
```

Comment:
// HandleUsersRequest

Reply:
Have you setup go tools on your vscode? it might catch these early on

Reply:
no will look into that thanks